package.json

```diff
@@ -101,6 +102,7 @@
     "ember-maybe-in-element": "^0.4.0",
     "ember-on-helper": "^0.1.0",
     "ember-popper": "^0.10.3",
+    "ember-style-modifier": "^0.5.0",
```

Comment:
👍 :wink:

Reply:
There is one downside of it: The style is not applied on server-side rendering / FastBoot. I'm planing to support it but it required more work as expected upstream at ember-cli-content-security-policy. I don't think this is an issue for this specific use case so. But wanted to make sure that we are aware.

Reply:
Yeah, understood. But as we (you!) were already using CSSOM API, triggered mostly in `didInsertElement`, and not in-template style assignments, we were not applying these things in FastBoot anyway.

Reply:
Yeah. Server-side rendering, dynamic styles and Content Security Policy is a tricky combination. When doing the changes I fear I missed that this might break styles in FastBoot. Glad that no one reported bugs so far. Cause I fear that there is currently no solution which does not violate a strict CSP but supports server-side rendering.